frontend/src/components/GemMap.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div>
+    <h4>Hello Friendos</h4>
+    Latitude: {{ currPosition.lat.toFixed(2) }}, Longitude:
+    {{ currPosition.lng.toFixed(2) }}
+  </div>
+
+  <google-map-loader
+    :mapConfig="mapConfig"
+    :apiKey="apiKey"
+    :markersOptions="markersOptions"
+  />
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, PropType } from "vue";
+import { useGeolocation } from "../useGeolocation";
+import GoogleMapLoader from "./GoogleMapLoader.vue";
+
+const DEFAULT_MAP_CONFIG = {
+  center: { lat: 1.3521, lng: 103.8198 },
```

Comment:
Nice default choice 😄 Should we set the center of the map after getting user coords?

Reply:
sounds good, done!